### PR TITLE
Metaprofile for Straumurinn ( xroad in Iceland )

### PR DESCRIPTION
**Pull request by samueljon**
This is a bare minimum metapackage that provides xroad-opmonitor as added dependency. No additional configuration changes at this time.

### CODE REVIEW

File: src/packages/src/xroad/redhat/SPECS/xroad-securityserver-is.spec [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+BuildArch:          noarch
+Group:              Applications/Internet
+License:            MIT
+Requires:           xroad-securityserver = %version-%release, xroad-addon-opmonitoring = %version-%release
+Conflicts:          xroad-centralserver
+
+%define src %{_topdir}/..
+
+%description
+This is meta package of X-Road security server with Finnish settings
```
Comment (iluwatar) [inline src/packages/src/xroad/redhat/SPECS/xroad-securityserver-is.spec]: Should be "Icelandic settings"

File: src/packages/src/xroad/ubuntu/generic/control [hunk 2]
```diff
@@ -140,6 +140,13 @@ Depends: xroad-securityserver (=${binary:Version}), xroad-addon-opmonitoring (=$
 Description: X-Road security server with Finnish settings
  This is meta-package of X-Road security server with Finnish settings.
 
+Package: xroad-securityserver-is
+Conflicts: xroad-centralserver
+Architecture: all
+Depends: xroad-securityserver (=${binary:Version}), xroad-addon-opmonitoring (=${binary:Version})
+Description: X-Road security server with Icelandic settings
+ This is meta-package of X-Road security server with Finnish settings.
```
Comment (iluwatar) [inline src/packages/src/xroad/ubuntu/generic/control]: Here too, should say "Icelandic settings" in multiple places

**Comment by samueljon** [comment]
I have updated the changes requested @iluwatar

File: src/packages/src/xroad/redhat/SPECS/xroad-securityserver-is.spec (hunk 1 above)
Comment (samueljon) [reply src/packages/src/xroad/redhat/SPECS/xroad-securityserver-is.spec]: fixed.

File: src/packages/src/xroad/ubuntu/generic/control (hunk 2 above)
Comment (samueljon) [reply src/packages/src/xroad/ubuntu/generic/control]: Fixed

**Pull request merged**